libs/hardhat/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { extendConfig } from "hardhat/config"
+import { HardhatConfig, HardhatUserConfig } from "hardhat/types"
+
+import "hardhat-dependency-compiler"
+import "@nomiclabs/hardhat-ethers"
+import "./tasks/deploy-zk-groups"
+import "./tasks/deploy-zk-groups-semaphore"
+
+extendConfig(
+    (config: HardhatConfig, userConfig: Readonly<HardhatUserConfig>) => {
+        config.dependencyCompiler.paths = [
+            "@semaphore-protocol/contracts/base/Pairing.sol",
```

Comment:
I haven't tested this, but I think we need to point to the ZkGroups solidity files here

Reply:
Yes, I think we need `"@zk-groups/contracts/ZKGroups.sol"`. Let me know as soon as you test it